content/src/view.js

```diff
@@ -1336,8 +1375,18 @@ function updatePaneLinks(pane) {
   }
   list = state.pane[pane].links;
   if (!list) { return; }
-  $('#' + pane).html('');
-  directory = $('<div class="directory"></div>').appendTo('#' + pane);
+
+  if ($('#fileSearch').length==0) {
+    $('<div class="search-file"><input type="text" id="fileSearch" style="width:100%" ></div>').appendTo('#' + pane);
+  }
+
+  if ($('#directory').length==0) {
+    directory = $('<div id="directory" class="directory"></div>').appendTo('#' + pane);
+  } else {
+      directory=$('#directory');
```

Comment:
space around operator `=`, and above, around `==`.

Thanks for your help on formatting.  Consistent formatting makes it easier to grep through code, and for similar reasons it also makes it easier for people to scan through code quickly.